### PR TITLE
Okay, fargate [cluster, tasks, services] are all named correctly

### PR DESCRIPTION
**Pull request by wimax-grapl**
### Which issue does this PR correspond to?
None

### What changes does this PR make to Grapl? Why?
Previously these were all autogenerated garbage; this is perhaps overly verbose but much nicer.
image

### How were these changes tested?
see that above image? :')

### CODE REVIEW

File: src/js/grapl-cdk/lib/fargate_service.ts [hunk 1]
```diff
@@ -64,8 +64,9 @@ export class FargateService {
 
     constructor(scope: cdk.Construct, serviceName: string, props: FargateServiceProps) {
         this.serviceName = `${props.prefix}-${serviceName}`;
-        const cluster = new ecs.Cluster(scope, `${this.serviceName}Cluster`, {
+        const cluster = new ecs.Cluster(scope, `${this.serviceName}-cluster`, {
             vpc: props.vpc,
+            clusterName: `${this.serviceName}-cluster`,
```
Comment (wimax-grapl) [inline src/js/grapl-cdk/lib/fargate_service.ts:69]: special sauce 1: get cluster named correctly

File: src/js/grapl-cdk/lib/fargate_service.ts [hunk 2]
```diff
@@ -90,8 +91,10 @@ export class FargateService {
         // Create a load-balanced Fargate service and make it public
         this.service = new ecs_patterns.QueueProcessingFargateService(
             scope,
-            `${props.prefix}-${serviceName}Service`, {
+            `${this.serviceName}-service`, {
             cluster,
+            serviceName: this.serviceName,
+            family: `${this.serviceName}-task`,
```
Comment (wimax-grapl) [inline src/js/grapl-cdk/lib/fargate_service.ts:97]: special sauce 2: get the TaskDefinition named correctly

File: src/js/grapl-cdk/lib/fargate_service.ts [hunk 3]
```diff
@@ -103,16 +106,17 @@ export class FargateService {
...
-            serviceName,
             cpu: 256,
             memoryLimitMiB: 512,
             desiredTaskCount: 1,
         });
 
         this.retryService = new ecs_patterns.QueueProcessingFargateService(
             scope,
-            `${props.prefix}-${serviceName}RetryService`, {
+            `${this.serviceName}RetryService`, {
```
Comment (jgrillo-grapl) [inline src/js/grapl-cdk/lib/fargate_service.ts:116]: should this get a hyphen too? E.g. `-retry-service`

**Review by jgrillo-grapl — APPROVED** [review]
LGTM, one question

**Review by ajm-grapl — APPROVED** [review]

**Pull request merged**